pkg/volume/projected/projected.go

```diff
@@ -350,7 +352,12 @@ func (c *projectedVolumeUnmounter) TearDownAt(dir string) error {
 	if err != nil {
 		return err
 	}
-	return wrapped.TearDownAt(dir)
+	if err = wrapped.TearDownAt(dir); err != nil {
```

Comment:
How can this fail?

Reply:
@mikedanese  
Maybe the dir parameter does not exist, or  the dir we want to tear down here is used by other processes.
I think we need to make sure the projected volume itself has been removed successfully, after that, go to clean tokens.